{
   "active_lock_reason" : null,
   "assignee" : null,
   "assignees" : [],
   "author_association" : "CONTRIBUTOR",
   "body" : "Saw this today both for legacy wallets\r\n\r\nhttps://cirrus-ci.com/task/5313927408517120?logs=ci#L3168\r\n\r\nand for descriptor wallets in another PR\r\n\r\nhttps://cirrus-ci.com/task/4911232080150528?logs=ci#L2755\r\n\r\n\r\n```\r\n test  2023-09-09T02:47:45.448000Z TestFramework (ERROR): Assertion failed \r\n                                   Traceback (most recent call last):\r\n                                     File \"/ci_container_base/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/test_framework.py\", line 131, in main\r\n                                       self.run_test()\r\n                                     File \"/ci_container_base/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/wallet_fundrawtransaction.py\", line 148, in run_test\r\n                                       self.test_external_inputs()\r\n                                     File \"/ci_container_base/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/wallet_fundrawtransaction.py\", line 1070, in test_external_inputs\r\n                                       assert_equal(self.nodes[0].testmempoolaccept([signed_tx[\"hex\"]])[0][\"allowed\"], True)\r\n                                     File \"/ci_container_base/ci/scratch/build/bitcoin-i686-pc-linux-gnu/test/functional/test_framework/util.py\", line 57, in assert_equal\r\n                                       raise AssertionError(\"not(%s)\" % \" == \".join(str(arg) for arg in (thing1, thing2) + args))\r\n                                   AssertionError: not(False == True)\r\n```\r\n\r\nBoth in `test/functional/wallet_fundrawtransaction.py`, line 1070:\r\n\r\n```python3\r\nassert_equal(self.nodes[0].testmempoolaccept([signed_tx[\"hex\"]])[0][\"allowed\"], True)\r\n```\r\n",
   "closed_at" : "2023-10-09T08:53:13Z",
   "closed_by" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=4",
      "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
      "followers_url" : "https://api.github.com/users/fanquake/followers",
      "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
      "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/fanquake",
      "id" : 863730,
      "login" : "fanquake",
      "node_id" : "MDQ6VXNlcjg2MzczMA==",
      "organizations_url" : "https://api.github.com/users/fanquake/orgs",
      "received_events_url" : "https://api.github.com/users/fanquake/received_events",
      "repos_url" : "https://api.github.com/users/fanquake/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/fanquake"
   },
   "comments" : 4,
   "comments_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437/comments",
   "created_at" : "2023-09-09T03:00:48Z",
   "events_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437/events",
   "html_url" : "https://github.com/bitcoin/bitcoin/issues/28437",
   "id" : 1888551001,
   "labels" : [
      {
         "color" : "d4c5f9",
         "default" : false,
         "description" : null,
         "id" : 62963516,
         "name" : "Tests",
         "node_id" : "MDU6TGFiZWw2Mjk2MzUxNg==",
         "url" : "https://api.github.com/repos/bitcoin/bitcoin/labels/Tests"
      }
   ],
   "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437/labels{/name}",
   "locked" : false,
   "milestone" : {
      "closed_at" : null,
      "closed_issues" : 69,
      "created_at" : "2023-04-15T11:36:05Z",
      "creator" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=4",
         "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
         "followers_url" : "https://api.github.com/users/fanquake/followers",
         "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
         "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/fanquake",
         "id" : 863730,
         "login" : "fanquake",
         "node_id" : "MDQ6VXNlcjg2MzczMA==",
         "organizations_url" : "https://api.github.com/users/fanquake/orgs",
         "received_events_url" : "https://api.github.com/users/fanquake/received_events",
         "repos_url" : "https://api.github.com/users/fanquake/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/fanquake"
      },
      "description" : "",
      "due_on" : null,
      "html_url" : "https://github.com/bitcoin/bitcoin/milestone/60",
      "id" : 9286970,
      "labels_url" : "https://api.github.com/repos/bitcoin/bitcoin/milestones/60/labels",
      "node_id" : "MI_kwDOABII584AjbU6",
      "number" : 60,
      "open_issues" : 11,
      "state" : "open",
      "title" : "26.0",
      "updated_at" : "2023-10-09T08:53:13Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/milestones/60"
   },
   "node_id" : "I_kwDOABII585wkQBZ",
   "number" : 28437,
   "performed_via_github_app" : null,
   "reactions" : {
      "+1" : 0,
      "-1" : 0,
      "confused" : 0,
      "eyes" : 0,
      "heart" : 0,
      "hooray" : 0,
      "laugh" : 0,
      "rocket" : 0,
      "total_count" : 0,
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437/reactions"
   },
   "repository_url" : "https://api.github.com/repos/bitcoin/bitcoin",
   "state" : "closed",
   "state_reason" : "completed",
   "timeline_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437/timeline",
   "title" : "testmempoolaccept CI failure in wallet_fundrawtransaction.py",
   "updated_at" : "2023-10-09T08:53:13Z",
   "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/28437",
   "user" : {
      "avatar_url" : "https://avatars.githubusercontent.com/u/2415484?v=4",
      "events_url" : "https://api.github.com/users/jonatack/events{/privacy}",
      "followers_url" : "https://api.github.com/users/jonatack/followers",
      "following_url" : "https://api.github.com/users/jonatack/following{/other_user}",
      "gists_url" : "https://api.github.com/users/jonatack/gists{/gist_id}",
      "gravatar_id" : "",
      "html_url" : "https://github.com/jonatack",
      "id" : 2415484,
      "login" : "jonatack",
      "node_id" : "MDQ6VXNlcjI0MTU0ODQ=",
      "organizations_url" : "https://api.github.com/users/jonatack/orgs",
      "received_events_url" : "https://api.github.com/users/jonatack/received_events",
      "repos_url" : "https://api.github.com/users/jonatack/repos",
      "site_admin" : false,
      "starred_url" : "https://api.github.com/users/jonatack/starred{/owner}{/repo}",
      "subscriptions_url" : "https://api.github.com/users/jonatack/subscriptions",
      "type" : "User",
      "url" : "https://api.github.com/users/jonatack"
   }
}
